### PR TITLE
docs Update readme file with configuration for Docker

### PR DESCRIPTION
**Pull request by vrulevskyi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,6 +134,21 @@ Analyzer is installed. For example, on Windows the default installation
 location is C:\\Program
 Files\\Fortify\\Fortify\_SCA\_and\_Apps\_*\<version\>*.
 
+### Configuration when running Jenkins from the Docker
```
Comment (hiro0824) [inline README.md:137]: Question:
Is the following description generated text or you made? If you made, I consulted our doc owner Karene, and she provided the following.

File: README.md [hunk 2]
```diff
@@ -134,6 +134,21 @@ Analyzer is installed. For example, on Windows the default installation
...
+
+When running Jenkins from under the Docker container you need to mount *\<sca\_install\_dir\>*
```
Comment (hiro0824) [inline README.md:139]: 'under the Docker container you need to mount' -> 'a Docker container, mount'

File: README.md [hunk 3]
```diff
@@ -134,6 +134,21 @@ Analyzer is installed. For example, on Windows the default installation
 location is C:\\Program
 Files\\Fortify\\Fortify\_SCA\_and\_Apps\_*\<version\>*.
 
+### Configuration when running Jenkins from the Docker
+
+When running Jenkins from under the Docker container you need to mount *\<sca\_install\_dir\>* 
+directory to the Docker container in order to make Fortify Static Code Analyzer executables
```
Comment (hiro0824) [inline README.md:140]: 'in order to make' -> 'to make'

File: README.md [hunk 4]
```diff
@@ -134,6 +134,21 @@ Analyzer is installed. For example, on Windows the default installation
...
+accessible from under the Docker. For instance, next command does this:
+
+    docker container run \
+      -p 8080:8080 \
+      -v /home/admin/Fortify/Fortify_SCA_and_Apps_20.2.0:/var/jenkins_home/Fortify/Fortify_SCA_and_Apps_20.2.0 \
+      --name=jenkins \
+      jenkins/jenkins -d
+
+The corresponding FORTIFY\_HOME should have the value `/var/jenkins_home/Fortify/Fortify_SCA_and_Apps_20.2.0`
```
Comment (hiro0824) [inline README.md:149]: -> For this example, the value of FORTIFY\_HOME is ‘/var/Jenkins_home/Fortify/Fortify_SCA_and_Apps_20.2.0’.

**Review by hiro0824 — COMMENTED** [review]
Karene is wondering if she should add something about Docker to the Jenkins Plugin guide... The current Jenkins guide is located here: [https://www.microfocus.com/documentation/fortify-jenkins-plugin/](url).
Please advise.

File: README.md (hunk 1 above)
Comment (vrulevskyi) [reply README.md:137]: I made description by myself. Of course, opinions are welcome 😄

File: README.md (hunk 2 above)
Comment (vrulevskyi) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (vrulevskyi) [reply README.md]: Done.

File: README.md (hunk 4 above)
Comment (vrulevskyi) [reply README.md]: Done.

**Comment by vrulevskyi** [comment]
Basically, this readme file change describes how to configure Docker container to make SCA_and_Apps executables accessible for Fortify Jenkins plugin when Jenkins is being run in the Docker. But documentation you referred is about the installation, configuration and using of Fortify Jenkins plugin itself. I think that although configuration of Docker container and configuration of Fortify Jenkins plugin are different things, they depend on each other and without proper configuration of Docker container the configuration of Fortify Jenkins plugin will not work and that's why is worth mentioning in the documentation.  But I am actually not sure in which way should be conveyed. Maybe Karene knows better?

**Review by tps193 — APPROVED** [review]
Look ok for me.

**Comment by vrulevskyi** [comment]
Merged.

**Pull request merged**